store/tikv/latch/scheduler.go

```diff
@@ -84,7 +84,7 @@ func (scheduler *LatchesScheduler) Lock(startTS uint64, keys [][]byte) *Lock {
 }
 
 // UnLock unlocks a lock with commitTS.
-func (scheduler *LatchesScheduler) UnLock(lock *Lock, commitTS uint64) {
+func (scheduler *LatchesScheduler) UnLock(lock *Lock) {
```

Comment:
IMHO, this API design is worse than before, because the user may forget to call `lock.SetCommitTS` after UnLock, while previous API make a stronger contract.

Reply:
Both are ok for me. It's suggested by @coocood ,  maybe I could update it when you reach an agreement?